packages/builder/src/pages/builder/_layout.svelte

```diff
@@ -9,10 +9,31 @@
   $: hasAdminUser = $admin?.checklist?.adminUser?.checked
   $: tenantSet = $auth.tenantSet
   $: cloud = $admin.cloud
+  $: user = $auth.user
+
+  const validateTenantId = async () => {
+    // set the tenant from the url in the cloud
+    const tenantId = window.location.host.split(".")[0]
+
+    if (!tenantId.includes("localhost:")) {
+      // user doesn't have permission to access this tenant - kick them out
+      if (user && user.tenantId && user.tenantId !== tenantId) {
```

Comment:
NAB can be `user?.tenantId !== tenantId`

Reply:
Added in https://github.com/Budibase/budibase/pull/2801/commits/8de6cd35850973505a587733757a76d31b948e29